tests/README.md

```diff
@@ -1,10 +1,10 @@
 
 ## Preparations
 
-1. The following executables must be copied or generated or linked into these locations, `sync_diff_inspector` can be downloaded from [tidb-enterprise-tools-latest-linux-amd64](http://download.pingcap.org/tidb-enterprise-tools-latest-linux-amd64.tar.gz), `tidb-server` can be downloaded from [tidb-master-linux-amd64](https://download.pingcap.org/tidb-master-linux-amd64.tar.gz):
+1. The following executables must be copied or generated or linked into these locations.
 
-    * `bin/tidb-server`
-    * `bin/sync_diff_inspector`
+    * `bin/tidb-server` can be downloaded from [tidb-master-linux-amd64](https://download.pingcap.org/tidb-master-linux-amd64.tar.gz) or installed by [tiup](https://github.com/pingcap/tiup)
```

Comment:
I think after the installation, user still need to place the binary here. please address how to find the binary

Reply:
added